### PR TITLE
Typo fixes on comments/strings

### PR DESCRIPTION
**Pull request by Killia0**
My first PR contribution to TShock is spellcheck huh, frankly I don't know why but hey I could spare the time for this and caught some stuff.

> ?????? HAVE YOU UPDATED THE CHANGELOG? ??????

No. this is just typo fixes with a spellchecker and deserves no credit.

### CODE REVIEW

**Review by hakusaro — DISMISSED** [review]
Thank you so much!

> No. this is just typo fixes with a spellchecker and deserves no credit.

This is a lie! Please, in the future, update the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,9 +13,8 @@ This is the rolling changelog for TShock for Terraria. Use past tense when addin
   * If there is no section called "Upcoming changes" below this line, please add one with `## Upcoming changes` as the first line, and then a bulleted item directly after with the first change.
 
 ## Upcoming changes
+* Fix some typos that have been in the repository for over a lustrum. (@Killia0)
```
Comment (hakusaro) [inline CHANGELOG.md]: Can you add this chronologically after the commit from Stealownz?

**Review by hakusaro — CHANGES_REQUESTED** [review]
Minor issue with the changelog's timespace contunity, can you adjust?

File: CHANGELOG.md (hunk 1 above)
Comment (Killia0) [reply CHANGELOG.md]: You mean, before it? because currently it is after

Comment (hakusaro) [reply CHANGELOG.md]: @Killia0 no, the changelog is:
1. first thing that changed
2. second thing that changed
3. third thing that changed

not the reverse, sorry.

Comment (Killia0) [reply CHANGELOG.md]: Huh, I see. I notice now that I also removed the `## TShock 4.5.5` on accident, Fixed now.

**Review by hakusaro — APPROVED** [review]
lgtm_7

**Pull request merged**